explorer/syncstatus.go

```diff
@@ -65,19 +65,28 @@ func (exp *explorerUI) BeginSyncStatusUpdates(barLoad chan *dbtypes.ProgressBarL
 		return
 	}
 
+	// stopTimer allows safe exit of the goroutine that triggers periodic
+	// websocket progress update.
+	stopTimer := make(chan bool)
```

Comment:
Works for me, but generally want to use `chan struct{}` if the value is unused since it is optimized to a zero byte payload in the channel send.